### ISSUE TITLE
--fileoutput is not printing to file

### ISSUE THREAD

**Issue by caruccio** [robusta-dev/krr#180]
**Describe the bug**

Using `--fileoutput result.json` creates an empty file.

**To Reproduce**

```
kind create cluster
helm install monitoring prometheus-community/kube-prometheus-stack -n monitoring --create-namespace
python krr.py simple -f json --fileoutput result.json
```
The whole output is being printed to stdout. The file `result.json` is empty.

```
$ ls -l  result.json 
-rw-r--r-- 1 mateus mateus 0 dez 14 15:57 result.json

```

**Expected behavior**

**Screenshots**

**Are you interested in contributing a fix for this?**
Yes/no. If yes, we will provide guidance what parts of the code to modify and help you.

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by caruccio** [comment]
A simple workaround to have json saved into a file is to print from `^{` to `^}`:

```
$ python krr.py simple -f json | sed -ne '/^{/,/^}/p' | tr -d '\n'| jq -r > result.json 
```

`tr -d '\n'` is required in order to transform multiline-strings into single line.

**Comment by aantn** [comment]
Should be fixed in #218

Can you confirm please?

**Comment by caruccio** [comment]
Confirmed it's now working. Thanks a lot.
